accord-core/src/main/java/accord/local/Node.java

```diff
@@ -738,7 +725,10 @@ public void receive(Request request, Id from, ReplyContext replyContext)
         if (waitForEpoch > topology.epoch())
         {
             configService.fetchTopologyForEpoch(waitForEpoch);
-            topology().awaitEpoch(waitForEpoch).addCallback(() -> receive(request, from, replyContext));
+            topology().awaitEpoch(waitForEpoch).addCallback((ignored, failure) -> {
```

Comment:
TODO handle failure?

Reply:
How though? The agent already got notified of the timeout. We are going to drop the message, and when we drop messages we don't normally send a response.

The error handling for this dropped message will be somewhere else that relies on the message to be received to make progress and will already have its own timeout.

Reply:
I updated this to follow the pattern I used elsewhere to wrap the exception and pass it to the agent.